File: Module3/Module3a/challenge/challenge10.js
//Create a function that filters out negative numbers

let numbers = [2, 6, -8, 90, 18, -54, -33, 27, -29, 7, -4]

function newFunc(positiveNumbers) {
    return positiveNumbers > 0
}

let newNumbers = numbers.filter(newFunc)

console.log(newNumbers)

File: Module4/Module4C/Challenge/challenge10.py
numbers = [2, 6, -8, 90, 18, -54, -33, 27, -29, 7, -4]


def positive_numbers(number):
    return [number for number in numbers if number >= 0]


print(positive_numbers(numbers))


# Create a function that filters out negative numbers.
